website/events/templates/events/event.html

```diff
@@ -213,7 +213,7 @@ <h1 class="section-title">{{ event.title }}</h1>
                                     {% else %}
                                         {% if not event.registration_required %}
                                             {% if event.no_registration_message %}
-                                                {{ event.no_registration_message }}
+                                                {{ event.no_registration_message | bleach }}
```

Comment:
Maybe it would be good to add a help text to the field "this field also accepts html tags, such as `Ticket sales via <a href="https://example.com" target="_blank">https://example.com</a>`